packages/salesforcedx-apex-debugger/src/core/sessionService.ts

```diff
@@ -13,11 +13,11 @@ import {
 import { RequestService } from '@salesforce/salesforcedx-utils-vscode/out/src/requestService';
 
 export class SessionService {
-  private userFilter: string;
-  private requestFilter: string;
-  private entryFilter: string;
-  private project: string;
-  private sessionId: string;
+  private userFilter?: string;
```

Comment:
@dehru - Were you able to check what @ntotten said and whether our tslint rules could have suggested to use `userFilter?` instead of `userFildter: string | undefined`?

Reply:
I do not see that option for tslint in the core rules.